config/config_test.go

```diff
@@ -431,6 +442,15 @@ experiment:
             signing_timeout_ms: 10
 hooks:
     enabled: true
+price_floors:
+    enabled: true
+account_defaults:
+    price_floors:
+        enabled: true
+        enforce_floors_rate: 100
+        adjust_for_bid_adjustment: true
```

Comment:
Can we set these to something other than the default values, say 0 and false, since the defaults are tested elsewhere?

Reply:
by default floors feature is disabled and in `TestFullConfig`  test feature is enabled.

Reply:
Ah my suggestion here is that we should set fields in `fullConfig` to something other than their default values, which you're doing for all of the floors fields except `enforce_floors_rate` and `adjust_for_bid_adjustment`. The default values for those two fields are 100 and true respectively and we test that the defaults are correctly applied in the config test `TestDefaults`. This test data structure `fullConfig` may be used to verify that values other than the default take precedence over the default values which is the reason for my suggestion.

Reply:
Updated test case to test  `enforce_floors_rate` and `adjust_for_bid_adjustment` for values other than default values